tests/shopify_python/test_google_styleguide.py

```diff
@@ -225,3 +225,51 @@ def test_use_cond_exprs(self):
         # root_not_msg should add no messages
         with self.assertNoMessages():
             self.walk(root_not_msg)
+
+    def test_unary_lambda_func(self):
+        unary_root = astroid.builder.parse("""
+        def unaryfnc():
+            unary_pass = lambda x: not (x+3)
+            unary_msg = lambda x: -x
```

Comment:
As a more realistic example, can you re-write these as `some_var = map(operator.not_, iterable)`